cirq/work/sampler.py

```diff
@@ -133,6 +175,133 @@ def sample(
 
         return pd.concat(results)
 
+    def sample_expectation_values(
+        self,
+        program: 'cirq.Circuit',
+        observables: Union['cirq.PauliSumLike', List['cirq.PauliSumLike']],
+        *,
+        num_samples: int,
+        params: 'cirq.Sweepable' = None,
+        permit_terminal_measurements: bool = False,
+    ) -> List[List[float]]:
+        """Calculates estimated expectation values from samples of a circuit.
+
+        This is a minimal implementation for measuring observables, and is best
+        reserved for simple use cases. For more complex use cases, consider
+        upgrading to `cirq/work/observable_measurement.py`. Additional features
+        provided by that toolkit include:
+            - Chunking of submissions to support more than (max_shots) from
+              Quantum Engine
+            - Checkpointing so you don't lose your work halfway through a job
+            - Measuring to a variance tolerance rather than a pre-specified
+              number of repetitions
+            - Readout error symmetrization and mitigation
+
+        This method can be run on any device or simulator that supports circuit
```

Comment:
Consider reordering or clarifying this.   It's unclear which "this method" you are referring to (observable measurement or sample_expectation_values).  Maybe consider putting the pointing to observable measurement at the end.  I think that makes more sense to explain what _this_ function does before pointing out alternatives.

Reply:
Clarified which method this refers to. I prefer to keep this paragraph at the top of the docstring, since all but the simplest use cases can benefit from using the Observable Measurement toolkit instead.

To that end, there's an argument to be made for dropping this API altogether: it doesn't provide functionality outside what will be possible with the Observable Measurement toolkit, and potentially distracts users from the "correct" tools for getting expectation values. As it stands, it primarily serves as a simple reference implementation, as pointed out above.